examples/model_selection/plot_precision_recall.py

```diff
@@ -7,55 +7,54 @@
 
 Precision-Recall is a useful measure of success of prediction when the
 classes are very imbalanced. In information retrieval, precision is a
-measure of result relevancy, while recall is a measure of how many truly
-relevant results are returned.
-
-The precision-recall curve shows the tradeoff between precision and
-recall for different threshold. A high area under the curve represents
-both high recall and high precision, where high precision relates to a
-low false positive rate, and high recall relates to a low false negative
-rate. High scores for both show that the classifier is returning accurate
-results (high precision), as well as returning a majority of all positive
-results (high recall).
-
-A system with high recall but low precision returns many results, but most of
-its predicted labels are incorrect when compared to the training labels. A
-system with high precision but low recall is just the opposite, returning very
-few results, but most of its predicted labels are correct when compared to the
-training labels. An ideal system with high precision and high recall will
-return many results, with all results labeled correctly.
+measure of the fraction of relevant items among actually returned items while recall
+is a measure of the fraction of items that were returned among all items that should
+have been returned. 'Relevancy' here refers to items that are
+postively labeled, i.e., true positives and false negatives.
 
 Precision (:math:`P`) is defined as the number of true positives (:math:`T_p`)
 over the number of true positives plus the number of false positives
 (:math:`F_p`).
 
-:math:`P = \\frac{T_p}{T_p+F_p}`
+.. math::
+    P = \\frac{T_p}{T_p+F_p}
 
 Recall (:math:`R`) is defined as the number of true positives (:math:`T_p`)
 over the number of true positives plus the number of false negatives
 (:math:`F_n`).
 
-:math:`R = \\frac{T_p}{T_p + F_n}`
+.. math::
+    R = \\frac{T_p}{T_p + F_n}
 
-These quantities are also related to the :math:`F_1` score, which is the
-harmonic mean of precision and recall. Thus, we can compute the :math:`F_1`
-using the following formula:
+The precision-recall curve shows the tradeoff between precision and
+recall for different thresholds. A high area under the curve represents
+both high recall and high precision, where high precision relates to low
+false positives in returned results, and high recall relates to a low false negatives
+in relevant results. High scores for both show that the classifier is returning
```

Comment:
I think "fewer" is more gramatically correct in this context than "low" false positives/negatives. What do you think of a phrasing:
> High precision corresponds to fewer false positives in returned results, and high recall corresponds to fewer false negatives in relevant results.

I also feel that the word "relates" is a bit vague. We can alternatively say something similar to:
> High precision can be achieved by having few false positives in the returned results, and high recall can achieved by having few false negatives in the relevant results.

Reply:
Sorry for delay. Amended, thanks! @ArturoAmorQ